### ISSUE TITLE
Can't Set Device for Volume Control on Harmony 665

### ISSUE THREAD

**Issue by RedSoxFan04** [congruity/congruity#4]
I bought a Harmony 665 a few days ago to replace my failing Harmony 350 from a couple years ago. I set it up on my MacBook Pro, but I ran into a problem - it turns out that the version of the Harmony software on my MacBook is too old for this remote. My MacBook is from 2011 and it can only go as far as macOS 10.13 (High Sierra), but the new Harmony software needs macOS Catalina (10.15) or later.

The version of the Harmony software on my MacBook was able to set up the remote fairly well, but with one glaring omission - volume controls. Towards the beginning of the setup process, it should have asked me which device should be used for volume control (i.e. TV, receiver), but it didn't. I looked into the matter and there's supposed to be another way to go about it, but that didn't work either.

I much prefer Linux and Congruity. Unfortunately, Congruity is missing a couple things, so I sometimes use Logitech's software.

I tried fixing this volume issue in Congruity, but no luck.

**Comment by swt2c** [comment]
Did you try the Setup Activities section of MHGUI?  In there you should be able to choose a volume control device I believe.

**Comment by RedSoxFan04** [comment]
Yeah, I did that with Logitech's software, but it doesn't totally solve the problem. It works if you're only using the Activity buttons, but what if you want to use a different device? What if you watch a TV show on your streaming box and then switch over to your Blu-Ray player or your Kodi box?

**Comment by swt2c** [comment]
I think that in that case you would set up a different activity for each of those things, e.g. a Watch Streaming Box activity, a Watch Blu-Ray activity, Watch Kodi activity, etc.

**Comment by RedSoxFan04** [comment]
Right, but what if you wanted to do it without turning the TV and A/V receiver on and off every time you wanted to switch devices? Or does it not work that  way? I have to admit, I haven't really experimented with it.

**Comment by swt2c** [comment]
If the TV and AV receiver are both part of Activity A and Activity B, and you switch from Activity A to Activity B, the remote won't turn off the TV and AV receiver.
